### PR TITLE
Sb-82 [BE] Allow speed to be post as null for heartbeat table

### PR DESCRIPTION
**Pull request by StudentCZ**
https://virgl.atlassian.net/jira/software/projects/SB/boards/1?selectedIssue=SB-82

After speaking with the frontend team regarding how speed should be handle, I got rid of the conditional statement that won't allow speed to be null.

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -70,7 +70,6 @@ def heartbeatpost():
         or not time_stamp
         or not latitude
         or not longitude
-        or not speed
     ):
         return "Unable to write to server due to missing attribute(s)", 400
```
Comment (chunloy) [inline app.py:75]: Please add .DS_Store to .gitignore. This file gets auto-generated from Mac computers when you access the project folder using Finder.

File: app.py [hunk 2]
```diff
@@ -70,7 +70,6 @@ def heartbeatpost():
         or not time_stamp
         or not latitude
         or not longitude
-        or not speed
```
Comment (chunloy) [inline app.py:73]: Could you please confirm that the POST request still works? On my end, I can't seem to make the POST request work in Postman. I'd expect to see speed = None in the local DB/GET request if it was received from the frontend as null.

**Review by chunloy — CHANGES_REQUESTED** [review]

File: app.py (hunk 2 above)
Comment (StudentCZ) [reply app.py:73]: I think something changed in the frontend that prevent the post request from working. I will look more into it with the Frontend team

File: app.py (hunk 1 above)
Comment (StudentCZ) [reply app.py:75]: Added .DS_Store to gitignore.

**Review by tonyxuan2021 — APPROVED** [review]
tested with both 0 and null for speed using postman, can no error was given. Also tested the server, still running after this change. Approved.

File: app.py (hunk 2 above)
Comment (StudentCZ) [reply app.py:73]: Actually nevermind, I just tested it again with Tony and we don't really need to frontend to be working for postman. We were able to send a post request to the heartbeat database with no errors when speed is 0 or null.

**Review by chunloy — APPROVED** [review]

**Pull request merged**